src/CPUSolver.cpp

```diff
@@ -530,13 +530,13 @@ void CPUSolver::transportSweep() {
   if (_cmfd != NULL && _cmfd->isFluxUpdateOn())
     _cmfd->zeroSurfaceCurrents();
 
-  /* Loop over azimuthal angle halfspaces */
-  for (int i=0; i < 2; i++) {
+  /* Loop over azimuthal angle and periodic track halfspaces */
+  for (int i=0; i < 4; i++) {
```

Comment:
Is this specific to 2D or generalization to 3D? If it's specific to 2D I would like to see a variable rather than the number 4 to make everything more translatable to 3D.